packages/vsce/src/utils/PersistentStorage.ts

```diff
@@ -184,6 +194,15 @@ export class PersistentStorage {
     return this.mBundlePartSearchHistory;
   }
 
+  public getLastUsedRegion(): { regionName: string | null; cicsPlexName: string | null; profileName: string | null } {
+    return this.mlastUsedRegion as { regionName: string | null; cicsPlexName: string | null; profileName: string | null };
```

Comment:
This is going to need an interface at some point (if there isn't already?) as we'll be using this object shape all over the place!